clusters/app.ci/supplemental-ci-images/snyk-cli/snyk-cli.yaml

```
@@ -0,0 +1,41 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: snyk-cli
+  namespace: ci
+---
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  namespace: ci
+  name: snyk-cli
+spec:
+  successfulBuildsHistoryLimit: 1
+  failedBuildsHistoryLimit: 2
+  output:
+    to:
+      kind: ImageStreamTag
+      name: snyk-cli:latest
```

nit: not sure if we need versioning on the tag. In case we do, we need to modify the image in the jobs as well.

```suggestion
      name: snyk-cli:go1.18
```

The choice is yours.